### PR TITLE
Clean up descriptions

### PR DESCRIPTION
**Pull request by CalvinScott**
Some small typos were found in the descriptions pre and post deployment. Quick fix to avoid embarrassment.

### CODE REVIEW

**Comment by CalvinScott** [comment]
@kregensberg @RongxinZhang Quick thumbs if you'd please

**Comment by RongxinZhang** [comment]
:v: looks good.

File: lib/listing-desc.js [hunk 1]
```diff
@@ -82,14 +82,14 @@ module.exports = {
   openParking: 'A boolean indicating if there is open space parking.',
   laundry: 'List of features and locations where the laundry is located in the property.',
   type: 'A list of types of properties such as Residential, Lease, Income, Land, Mobile, etc.',
-  subtype: 'A list of types of residential and residential lease properties, i.e. SFR, Condo',
+  subtype: 'A list of types of residential and residential lease properties, i.e. SFR, Condo, etc.',
```
Comment (kregensberg) [inline lib/listing-desc.js]: should be e.g., not i.e.

**Comment by kregensberg** [comment]
Just one more: 
-   yearBuiltDetails: 'A description of the details _behing_ the year the structure was built.',

**Comment by CalvinScott** [comment]
@kregensberg ping pong

**Comment by kregensberg** [comment]
:+1:

**Pull request merged**